config/locales/en.yml

```diff
@@ -498,55 +500,56 @@ en:
-      header:
-        date_calculation: Date Calculation of Draft Batch
-        day_select: Which day of the week do you plan to publish your episodes?
-        week_select: Which weeks of each month do you plan to publish on?
-        range_select: How long will episodes recur?
+      title: Plan Episodes
+    form_calculator_dates:
+      help:
+        selected_days: Which day of the week do you plan to publish your episodes? Select all that apply.
+        selected_weeks: Which weeks of the month do you plan to publish? Select all that apply.
+        start_date: How long should episodes recur? Give a date range with a start date, and choose how to calculate the end date.
+      label:
+        end_date: End Date
+        number_of_episodes: Number of Episodes
+        or: OR
+        selected_days: Days of Week
+        selected_weeks: Weeks of Month
+        start_date: Start Date
+    form_draft:
       help:
-        date_calculation: Calculate release dates for your drafts by selecting which days and weeks you plan to publish, and then select a date range.
-        day_select: Which day of the week do you plan to publish your episodes? Select all that apply.
-        week_select: Which weeks of the month do you plan to publish? Select all that apply.
-        calculate_weeks: Calculate the weeks by
-        select_weeks: Select weeks
-        select_period: Select interval
-        range_select: How long should episodes recur? Give a date range with a start date, and choose how to calculate the end date.
-        calculate_range: Calculate the end of the batch by
-        select_range: Select range
+        settings: Enter the number of ad-breaks in your episode (if applicable), and the time of day you plan to release your episodes.
       label:
-        week_condition: Calculate the weeks by
-        week_of_month: Select Week of the month
-        period: Select weekly interval
-        date_range_condition: Calculate the end of the batch by
-    form_side:
-      header:
-        preview: Preview Draft Schedule
-        publish_time: What time of day do you plan to publish the episodes?
-        audio_segments: How many audio segments?
-        segment_count: Number of segments
-        number_of_drafts: "Drafts planned: "
-        help:
-          segment_count: Enter the number of original audio segments. The number of segments helps determine the number and location of ad breaks for this episode.
+        create: Create
+        draft: Draft
+        key: Key
+        key_duplicate: Duplicate Episode Drafts
+        key_existing: Existing Episode Draft
+        key_new: New Episode Draft
+        month: Month
+      title:
+        calendar: Production Calendar
+        settings: Draft Settings
+    form_draft_settings:
       help:
-        preview: Select more options to get a schedule preview.
+        ad_breaks: Do we need this?
+        medium: Do we need this?
+        publish_time: Do we need this?
```

Comment:
I ported a bunch of the text from the previous version into help icons/popovers.  But probably need to audit what's here, and what's useful.

Reply:
verbiage for these labels? @brandonhundt